test/integration/http2_upstream_integration_test.cc

```diff
@@ -280,6 +280,15 @@ TEST_P(Http2UpstreamIntegrationTest, ManyLargeSimultaneousRequestWithBufferLimit
   manySimultaneousRequests(1024 * 20, 1024 * 20);
 }
 
+TEST_P(Http2UpstreamIntegrationTest, ManyLargeSimultaneousRequestWithRandomBackup) {
+  config_helper_.addFilter(R"EOF(
+  name: random-pause-filter
+  config: {}
+  )EOF");
+
+  manySimultaneousRequests(1024 * 20, 1024 * 20);
```

Comment:
If we are going to do a random test, should we do this multiple times? Perhaps 5 or 10? WDYT?

Reply:
I'm fine with it being fuzzed over many CI runs, but if you think it's worth it I can refactor all the initialization set up out of the function so we can do the requests multiple times.

Reply:
Up to you, if you think CI is enough that's fine.